src/ImageSharp/Formats/ImageExtensions.Save.cs

```diff
@@ -9,6 +9,7 @@
 using SixLabors.ImageSharp.Formats.Jpeg;
 using SixLabors.ImageSharp.Formats.Pbm;
 using SixLabors.ImageSharp.Formats.Png;
+using SixLabors.ImageSharp.Formats.Qoi;
```

Comment:
One last important thing I almost missed ...

This file is auto generated. If you change & save the `ImageExtensions.Save.tt` in Visual Studio, it will generate `ImageExtensions.Save.cs` for you. There is an extension for VS Code and Rider as well.

Wanted to push the necessary changes for you, but looks like I don't have the rights. You can just grab this commit if you want: https://github.com/SixLabors/ImageSharp/commit/2700f8b40e2841fad3e615380cc7213255727981

Reply:
Sorry, I only forked it :(

I'll add all the changes you told me in one commit because I'm not sure how to get that commit into my fork, so I'll add it manually but thanks